### ISSUE TITLE
[Accessibility] Custom keybindings do not work for "agentSessionsViewer.refresh" and "agentSessionsViewer.find"

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#307649]
CC @meganrogge

Tested on Windows. If you assign keybindings to the following commands, you would see the warning and the command isn't triggered:

> Warning: TypeError: Cannot read properties of undefined (reading 'openFind')

`keybindings.json`:

```json
[]
{
    "key": "f5",
    "command": "agentSessionsViewer.refresh",
    "when": "agentSessionsViewerFocused"
  },
  {
    "key": "ctrl+shift+alt+f",
    "command": "agentSessionsViewer.find"
  }
  ]
```

**Comment by meganrogge** [comment]
I can reproduce the issue

**Comment by bpasero** [comment]
@meganrogge could you help out
